components/byu-header/byu-header.js

```diff
@@ -1,52 +1,44 @@
-import { LitElement, html, customElement, property } from 'lit-element';
-import style from './byu-header.scss';
+import { html, css, customElement, LitElement, unsafeCSS } from 'lit-element'
+import style from './byu-header.sass'
 
 @customElement('byu-header')
 export class BYUHeader extends LitElement {
 
-  @property()
-  title = 'Site Title';
+  static get styles () {
+    return css`${unsafeCSS(style)}`
+  }
 
-  static styles = style;
-  render(){
+  render () {
     return html`
-   <header role="banner" class="byu-header">
-        <div class="byu-site-banner">
-
-            <a class="byu-link" href="https://byu.edu">
-                <span class="visually-hidden">BYU</span>
-            </a>
-
-            <div class="byu-titles">
-
-                <h1 class="byu-site-title">
-                    <slot id="site-title" name="site-title"><a href="/">${this.title}</a></slot>
-                </h1>
-
-                <button class="byu-menu-button" aria-expanded="false">Menu</button>
-
-            </div>
-
-            <div class="byu-action-id-search">
-
-                <form class="byu-search" action="/">
-                    <label class="byu-search-label" for="site-search">Search</label>
-                    <input type="search" id="byu-site-search" name="q" aria-label="Site search" placeholder="Search">
-                    <button class="byu-search-btn">
-                        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
-                            <style>.st{stroke-width:10;fill:none;stroke:currentColor;stroke-linecap:round;}</style>
-                            <circle class="st" cx="45.5" cy="45.5" r="24.5"/><path class="st" d="M63 63l16 16"/>
-                        </svg>
-                    </button>
-                </form>
-
+<header role="banner" class="byu-header">
+    <div class="byu-site-banner">
+        <a class="byu-link" href="https://byu.edu">
+            <span class="visually-hidden">BYU</span>
+        </a>
+        <div class="byu-titles">
+            <nav class="byu-breadcrumb">
```

Comment:
One thing we'll need to do eventually is allow multiple breadcrumbs. I don't think it needs to happen on this PR since we're still pre-release, but at some point before releasing we'll want to support scenarios like 

"College of Life Sciences | Biology Department" 

in the breadcrumb. I tried to get a screenshot but I couldn't find Nate's old header designs in Slack, so if that's unclear let me know. Like I said, it doesn't need to be in this PR, but that should be on the radar.

Reply:
Yes. I agree. I'll add a TODO.